tx-pool/src/component/edges.rs

```diff
@@ -28,6 +29,10 @@ impl Edges {
     }
 
     pub(crate) fn insert_input(&mut self, out_point: OutPoint, txid: ProposalShortId) {
+        // inputs is occupied means double speanding happened here
```

Comment:
I think this law change will have side effects. It will squeeze in late-entry transactions and destroy the behavior of data. We should not allow wrong behavior to continue.